### PR TITLE
ui & api. add cockpit interface

### PR DESCRIPTION
**Pull request by federicoballarini**
Refer to https://github.com/NethServer/dev/issues/6003

### CODE REVIEW

**Comment by federicoballarini** [comment]
Waiting for review: @gsanchietti @edospadoni. Thank you.

File: ui/src/views/Settings.vue [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+      <span class="pficon pficon-error-circle-o"></span>
+      {{ errorMessage }}.
+    </div>
+
+    <div v-if="!uiLoaded" class="spinner spinner-lg"></div>
+    <div v-if="uiLoaded">
+      <form class="form-horizontal" v-on:submit.prevent="saveConfig()">
+        <div class="row">
+          <div class="col-lg-12">
+            <div class="col-md-6">
```
Comment (edospadoni) [inline ui/src/views/Settings.vue]: You can remove this `md-6` class and modify all `col-sm-5` labels to `col-sm-2` to match other modules styles

**Comment by edospadoni** [comment]
The UI part is good :+1: . For the labels maybe @lucagasparini could help us.

File: ui/src/views/Settings.vue (hunk 1 above)
Comment (federicoballarini) [reply ui/src/views/Settings.vue]: Changed.

**Comment by federicoballarini** [comment]
@edospadoni This is the view on a MacBook Pro 13" (with normal zoom).
I think that we need to review fields size. We can leave `col-sm-2 contol-label`, but for input fields I think we should move them to `col-sm-4`.

What do you think?

Schermata 2020-01-07 alle 19 19 34

**Comment by federicoballarini** [comment]
And this is a view on a 21.5 monitor.

image

I leave you the choice...

**Comment by edospadoni** [comment]
Yes, the previous changes is only for labels, in the other part we can set `col`sm-5` as in other modules

**Comment by federicoballarini** [comment]
I think now it should be ok @edospadoni @gsanchietti

**Comment by edospadoni** [comment]
> I think now it should be ok @edospadoni @gsanchietti

Yes, now it's ok. Great job!! :+1:

**Comment by gsanchietti** [comment]
Would you mind removing the `.gitignore` file?
If you need such config, please keep it in you home environment :)

**Comment by federicoballarini** [comment]
> Would you mind removing the `.gitignore` file?
> If you need such config, please keep it in you home environment :)

For me it's not a problem. I copied it from another nethserver  package.

**Comment by gsanchietti** [comment]
Ok, that's fine, leave it as is ;)

**Pull request merged**